rsdp/src/handler.rs

```diff
@@ -10,8 +10,8 @@ where
 {
     physical_start: usize,
     virtual_start: NonNull<T>,
-    region_length: usize, // Can be equal or larger than size_of::<T>()
-    mapped_length: usize, // Differs from `region_length` if padding is added for alignment
+    region_length: usize, // Desired physical memory span, can be equal or larger than size_of::<T>()
+    mapped_length: usize, // Usuable mapped length starting from `virtual_start`
```

Comment:
Do you think documentation for these fields should be moved to `PhysicalMapping::new`? Its arguments are named the same as the fields, and the documentation under `PhysicalMapping::new` has the benefit that it is publicly facing.